tsconfig.json

```diff
@@ -4,6 +4,7 @@
   "compilerOptions": {
     "resolveJsonModule": true,
     "target": "es5",
+    "lib": ["es2019"],
```

Comment:
Pardon the ignorance but I guess if we're specifying ES version used in our code, why not ES2020 ;)?

Reply:
I moved to using `ESNext`